### PR TITLE
Merges the analyze air verb with toggle gas analyzer

### PR DESCRIPTION
**Pull request by r1ks-iwnl**
## What Does This PR Do
Merges the analyze air and toggle gas analyzer verbs 
Adds ability to analyze /obj/machinery/atmospherics/portable
Analyzing is now done with CtrlClickOn instead of attack_ghost(conflicting with the attack_ghost on /obj/machinery/atmospherics/portable stuff and afaik this seemed to be the simplest solution)

## Why It's Good For The Game
No reason for these 2 verbs to be separated, also seeing inside canisters, etc. is useful

## Testing
Analyzed stuff on local

## Changelog
:cl:
tweak: Merges the analyze air verb with toggle gas analyzer
tweak: You now use the analyze air verb with Ctrl+Click
/:cl:

### CODE REVIEW

File: code/_onclick/observer_onclick.dm [hunk 1]
```diff
@@ -39,12 +39,16 @@
 			M = get_mob_in_atom_with_warning(A)
 			if(M)
 				client.holder.show_player_panel(M)
+			CtrlClickOn(A)
```
Comment (Contrabang) [inline code/_onclick/observer_onclick.dm:42]: For admins, I suggest doing if they CANT find a mob
```suggestion
				client.holder.show_player_panel(M)
			else
				CtrlClickOn(A)
```

Comment (r1ks-iwnl) [reply code/_onclick/observer_onclick.dm:42]: Makes you unable to scan on the same tile as a mob. I could just check if you clicked on a mob directly but that would make it a bit harder to open the panel so maybe not.

Comment (Contrabang) [reply code/_onclick/observer_onclick.dm:42]: I think that its fine in adminmode you cant always scan pipes if someones standing over them

**Comment by larentoun** [comment]
https://github.com/ParadiseSS13/Paradise/pull/24975 also covers this

**Comment by r1ks-iwnl** [comment]
> #24975 also covers this

Well that's a billion times better

**Pull request closed without merging**